plumbing/protocol/packp/sideband/demux.go

```diff
@@ -126,13 +125,17 @@ func (d *Demuxer) nextPackData() ([]byte, error) {
 	case PackData:
 		return content[1:], nil
 	case ProgressMessage:
-		_, err := d.Progress.(io.Writer).Write(content[1:])
-		return nil, err
+		if d.Progress != nil {
```

Comment:
Is a optional argument, and not the most common behavior, so that's why is not part of the New method